jerry-core/lit/lit-globals.h

```diff
@@ -94,6 +94,16 @@ typedef ecma_char_t *ecma_char_ptr_t;
 #define LIT_UTF8_MAX_BYTES_IN_CODE_POINT (4)
```

Comment:
Sorry for the delay.
You are correct, CESU uses UTF-8 encoding, the only difference is with non-BMP code points. In that case we first separate the code point into a surrogate pair, and encode the surrogates separately in UTF-8, which means that these characters will be encoded in 6 bytes instead of 4. This may seem like a waste, but in this case we can directly decode the string into code units, without the need for special handling of non-BMP code points, which are very rare anyway.

We'll have to keep some of the UTF-8 support though, so I don't think that we should mass rename every definition.